quantum/quantum.c

```diff
@@ -459,9 +459,16 @@ void set_single_persistent_default_layer(uint8_t default_layer) {
 // Override these functions in your keymap file to play different tunes on
 // different events such as startup and bootloader jump
 
-__attribute__((weak)) void startup_user(void) {}
```

Comment:
Removed this hook by mistake, im assuming. It is still listed on `quantum.h:281`

Reply:
No, not by mistake. This hook isn't actually called anywhere except for custom code.  And keyboard_post_init_user takes care of this (as it is the absolute last call during init).   And where it is used, it is called by matrix_init_user, and it appears to be called solely for audio startup, which has long since been part of core.

But yeah, need to remove the prototype from the header file.